testing/combined-cs/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
   implementation(project(":nessie-versioned-storage-common"))
   implementation(project(":nessie-versioned-storage-inmemory"))
   implementation(project(":nessie-versioned-storage-store"))
+  implementation(libs.guava)
```

Comment:
Shall we move the combined client out of the `testing` folder?

Reply:
Want to do that, but in a separate PR.